cpu/stm32_common/periph/flashpage.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     cpu_stm32f0
+ * @ingroup     cpu_stm32_common
```

Comment:
I think it should appear clearly that it works only with stm32f0 and stm32f1.
Maybe adding a compile guard based on cpu family would be a good idea, too.